src/orion/core/cli/db/setup.py

```diff
@@ -53,11 +54,36 @@ def main(*args):
         if cancel.strip().lower() == "n":
             return
 
-    _type = ask_question("Enter the database type: ", "mongodb")
-    name = ask_question("Enter the database name: ", "test")
-    host = ask_question("Enter the database host: ", "localhost")
-
-    config = {"database": {"type": _type, "name": name, "host": host}}
+    # Get database type.
+    while True:
+        _type = (
```

Comment:
I think it would be great to push this loop into ask_question with a `choice` argument. If there is no choice, the loop breaks directly, otherwise it ensures the user input is valid. What do you think?

Reply:
Ok ! Done !